### PR TITLE
Update contributor page

### PR DESCRIPTION
**Pull request by samperson1997**
This PR will show more detailed information of committers and contributors in website page. Afterwards, if you want to maintain the information, here are the simple steps:
1. Search your name in `Contributor.vue`, and update the information (including avatar, name, role, organization, apache id, and starting date).
2. If your name hasn't existed in the page, you can just add your information in the list accordingly: `conList` for committers, `conWithPRList` for contributors who have submitted PR(s), and `conWithIssueList` for contributors who have proposed issue(s).

As for avatar, you can either upload a picture (better to limit the **size <150KB**) by adding one in `/assets/img/contributor-avatar`, then you only need to use the file name in the information:
```
avatar: 'xdh.jpeg',
name: 'Xiangdong Huang',
 ...
```
Or, you can use a link in the information:
```
avatar: 'https://avatars1.githubusercontent.com/u/19167280?s=400&u=b7fa4acd5b6161283946c2ecb5e4837b6f137a1b&v=4',
name: 'Zesong Sun',
...
```
Both photo file and picture link work.

### CODE REVIEW

**Comment by jixuan1989** [comment]
And I want to change a better avatar for me :) (no hurry, it does not block the acceptance of this PR)

File: src/views/Contributor.vue [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    data() {
+      return {
+        msg: 'Welcome to Community Page',
+        md: "",
+        toc: "",
+        conList: [{
+          avatar: 'http://ise.thss.tsinghua.edu.cn/~wangjianmin/jimwang.jpg',
+          name: 'Jianmin Wang',
+          role: 'Committer and PPMC member',
+          org: 'Tsinghua University'
```
[the remark this answers is not in the document]
Comment (samperson1997) [reply src/views/Contributor.vue]: Hi, there is `id` and `date` for those whose information is known by me 😂 (as you can see in the snapshot). Do you mean that there should be default "blank" info for those whose info is currently unknown?

[the remark this answers is not in the document]
Comment (sunjincheng121) [reply src/views/Contributor.vue]: Yes, I think is better to have a default value for the column which need update by himself. In this way may remind the committer to update, and I suggest using `Your Apache ID should be here` as default value here :)

What do you think?

**Comment by liutaohua** [comment]
I think although I am not as handsome as @jixuan1989 , I also want to update my photos 💯

File: src/views/Contributor.vue [hunk 2]
```diff
@@ -0,0 +1,215 @@
+<template>
+  <div>
+    <div class="container">
+      <div class="row markdown-body">
+        <div class="col-sm-8">
+          <h2>People</h2>
+          <h3>Committers</h3>
+          <SingleContributor v-for="(con,index) in conList" :key="'1' + index" :avatar="con.avatar" :name="con.name"
+                             :org="con.org" :role="con.role" :id="con.id" :date="con.date"></SingleContributor>
+          <h3>Contributors who submit PRs</h3>
```
Comment (sunjincheng121) [inline src/views/Contributor.vue]: I think without this part in this page is fine, as our community continues to grow, we do n’t seem to have the energy to merge updates from many new contributors a day, and we ca n’t ask every contributor to be willing to update this page, especially users who just report issues.  What do you think? :)

File: src/views/Contributor.vue [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+          <SingleContributor v-for="(con,index) in conWithPRList" :key="'2' + index" :avatar="con.avatar"
+                             :name="con.name"
+                             :org="con.org" :role="con.role" :id="con.id" :date="con.date"></SingleContributor>
+          <h3>Contributors who submit issues</h3>
```
Comment (sunjincheng121) [inline src/views/Contributor.vue]: ditto

**Comment by jixuan1989** [comment]
Hi Zesong,

As discussed on the mailing list, I think we can remove the company info (but if someone want to show his/her info, it is ok).

After that, we can sync the roster of committers.

**Comment by sunjincheng121** [comment]
@samperson1997 Could you please have look at the comments and I appreciate if you can update the PR. then we can merge it ASAP. :)

**Pull request merged**